frontend/src/metabase/models/components/NewModelOption.jsx

```diff
@@ -42,11 +42,14 @@ export default class NewModelOption extends Component {
           />
         </div>
         <div
-          className="text-normal mt2 mb2 text-paragraph"
+          className={cx(CS.textNormal, "mt2 mb2", CS.textParagraph)}
           style={{ lineHeight: "1.25em" }}
         >
           <h2 className={cx(S.modelTitle, { "text-brand": hover })}>{title}</h2>
-          <p className="text-medium text-small" style={{ maxWidth: "360px" }}>
+          <p
+            className={cx("text-medium", CS.textSmall)}
```

Comment:
text-medium is a color, text-small is a size 😭

Reply:
oh yes! 😂